modules/social_features/social_event/modules/social_event_enrollment_max/config/install/field.field.node.event.field_event_max_enroll.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_event_max_enroll
+    - node.type.event
+id: node.event.field_event_max_enroll
+field_name: field_event_max_enroll
+entity_type: node
+bundle: event
+label: 'Max participant number'
+description: 'Set a limit for event enrollments. Users will not be able to enrol once the max participant number will be reached.'
```

Comment:
Note, there doesn't seem to be a default value nor is it:
a) clear how to create an event and not set any mex enrolment limit.
b) clear what will happen if you leave it empty (is that possible or is it required?)